### PR TITLE
Sb 55 stake

### PR DESCRIPTION
**Pull request by fenrew**
Added stakes and everything that is needed for it

### CODE REVIEW

File: combat/combat.js [hunk 1]
```diff
@@ -84,12 +84,12 @@ const duelFullArmy = (user, opp) => {
 	const oppHp = oppStats.health * oModifier;
 	const oppAt = oppStats.attack * oModifier;
 
-	const losses = Math.floor((userHp + userAt) - (oppHp + oppAt));
+	const winMargin = Math.floor((userHp + userAt) - (oppHp + oppAt));
 
 	// Determine winner
-	const win = losses > 0 ? true : false;
+	const win = winMargin > 0 ? true : false;
```
Comment (Ojself) [inline combat/combat.js:90]: Ternary er ikke nødvendig her. Prøv:
``` const win = winMargin > 0 ```

File: game/_CONSTS/cooldowns.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const Discord = require("discord.js");
 // alphabetical
 const cooldowns = {
 	dailyPrice:(1000 * 60 * 60 * 24),
-	duel: (1000 * 60 * 2),
+	duel: (1000),
```
Comment (Ojself) [inline game/_CONSTS/cooldowns.js:7]: Dette er veldig lavt..?

File: combat/combat.js (hunk 1 above)
Comment (fenrew) [reply combat/combat.js:90]: Aaah, sant! Ty!

File: game/_CONSTS/cooldowns.js (hunk 2 above)
Comment (fenrew) [reply game/_CONSTS/cooldowns.js:7]: For testing purposes;) Det var egentlig 2 min, så jeg setter det tilbake

**Pull request merged**